### ISSUE TITLE
Bug: Missing gems in Mega Mode

### ISSUE THREAD

**Issue by thepry** [tongueroo/jets#93]
Hi, when I add en example project (http://rubyonjets.com/docs/rails-support/) to jets and deploy, I get error messages about missing gem. 

I managed to fix it by adding missing gems to the Gemfile in the jets project root.

```
gem 'nio4r'
gem 'websocket-driver', '0.6.5'
gem 'ffi', '1.9.25'
gem 'puma', '3.12.0'
```

Here is the example from cloudwatch
```
11:23:55
[0;32;49m=> cd ./rack && bin/rackup[0m

11:23:56
/var/runtime/gems/bundler-1.17.1/lib/bundler/spec_set.rb:91:in `block in materialize': Could not find nio4r-2.3.1 in any of the sources (Bundler::GemNotFound)
/var/runtime/gems/bundler-1.17.1/lib/bundler/spec_set.rb:91:in `block in materialize': Could not find nio4r-2.3.1 in any of the sources (Bundler::GemNotFound)

11:23:56
from /var/runtime/gems/bundler-1.17.1/lib/bundler/spec_set.rb:85:in `map!'
```

I'm using the latest version of jets.
```
jets -v
1.3.5

ruby -v
ruby 2.5.3p105
```

**Comment by tongueroo** [comment]
Interesting that your Gemfile locks down version for the other gems but didnt need to lock down the version for nio4r which the error reports: Could not find nio4r-2.3.1

In any case, the nio4r-2.3.1 gem is now available. Automated job actually took care of it.

**Comment by thepry** [comment]
@tongueroo It's at first nio4r. Then, if I add just it to the Gemfile the error would be in about `websocket-driver` or `ffi`

**Comment by tongueroo** [comment]
Ah I see. Make sense now. Thanks for explaining.

**Comment by thepry** [comment]
Hi. This error is still present and Megamode doesn't work until I add `nio4r` and other gems to the Gemfile in jets root. 

Otherwise I'm getting this in cloudwatch logs:

```
/var/runtime/gems/bundler-1.17.1/lib/bundler/spec_set.rb:91:in `block in materialize': Could not find nio4r-2.3.1 in any of the sources (Bundler::GemNotFound)
```

**Comment by tongueroo** [comment]
It was a bug gem build process.  Discovered also right before you opened the issue actually.

Fixed this in the latest v1.4.0 release [CHANGELOG](https://github.com/tongueroo/jets/blob/master/CHANGELOG.md).  Also introduced a much better [Rails Support](http://rubyonjets.com/docs/rails-support/), we can deploy from within the Rails app and Jets will auto-detect this now. Feel it's a much more simple flow.
